docs/users/globebrowsing/geojson-layers.md

```diff
@@ -0,0 +1,160 @@
+| **Points Specific**          |                   |                              |  |
+| PointSize           | `point-size`                     | float (> 0)                              | A value that will be used to decide the size of rendered points. The size will be scaled based on the bounding sphere of the globe   |
+| PointTexture        | `texture`, `sprite`, `point-texture` | string (path to a texture)           | A texture to be used for rendering points |
+| PointTextureAnchor  | `point-anchor`, `anchor`         | string (`"bottom"` or `"center"`)        | Decides the placement of the point texture in relation to the position. Default is a the bottom of the texture, but it can also be put at the center |
+
+### Other useful properties and settings
+In addition to the visual properties in the GeoJson files, some regular OpenSpace properties can be set for any loaded features. These can be used to navigate to a feature, change the visuals to match the projection system you are currently using, or just alter the look in general. These can not be set from a .geojson file, but through the Scene menu in the user interface, or in an asset.
+
+#### Height offset, opacity, and scale
+A few helper properties have been added to simplify changing some aspects of the visuals of the entire collection at once:
+
+- *Opacity* - change the opacity of all features in a collection
+- *Height Offset* - move the geometry up/down in relation to the reference surface (based on the height map or sea level, depending on chosen Altitude Mode)
+- *Point Size Scale* - increase/decrease the size of any points in the collection
+- *Line Width Scale* - increase/decrease the width of any rendered lines in the collection
+
+#### Alignment of points
+For points, it is possible to change how the texture planes align against the globe surface. There are four different options:
+
+| Option           | Value | Description |
+| ---------------- | :---: | ----------- |
+| Camera Direction | 0     | Rotate the plane to face the camera view direction and alight the up-driection with the camera's up. Suitable for planar displays |
+| Camera Position  | 1     | Use the camera's up direction, but rotate the plane to face the camera position instead of view direction. Suitable for spherical displays, such as domes or planetariums |
+| Globe Normal     | 2     | Use the out-driection of the globe as the up direction |
+| Globe Surface    | 3     | Align the plane flat along the globe surface |
+
+
+Here are examples of points aligned with the Camera Direction, Globe Normal and Globe Surface options, respectively. The points have been extruded with a line that intersects the globe surface
+![point alignment examples](./images/geojson/points_transparent.png)
+
+#### Light sources
+Polygon geometry can be lit using an OpenSpace light source (for example the Sun, or the Camera). Per default, a camera-based light source is added for all features.
```

Comment:
How would it look like to add a Sun light source instead? Maybe add an example with it or just a line or two about how that would look in the asset.

Reply:
Did not want to add an example until it's working well. Also, ideally I would want to have another page on light sources in general to refer to instead